FinnUI/Sources/MyVehicles/MyVehiclesListView/MyVehiclesListView.swift

```diff
@@ -0,0 +1,118 @@
+import SwiftUI
+
+@available(iOS 13.0.0, *)
+struct MyVehiclesListView: View {
+
+    // MARK: - Private
+
+    @State var navigateToPreviouslyOwnedVehicles = false
+    private let imageUrl = "https://images.finncdn.no/dynamic/default/"
```

Comment:
I am not sure if this should be an internal constant.

either a parameter or that each item contains the full url

Reply:
Agree. i'll just the whole thing into the `MyVehicleCell` as it belongs/needed there